src/systems/userinput/userinput.js

```diff
@@ -66,7 +66,7 @@ AFRAME.registerSystem("userinput", {
   init() {
     this.frame = {};
 
-    this.activeSets = new Set([sets.global]);
+    this.activeSets = new Set([sets.global, sets.globalPost]);
```

Comment:
Annoying that we had to add this, but we will be able to remove it soon with the work John is doing now on determining binding dependency